promitor-agent-resource-discovery/README.md

```diff
@@ -63,8 +63,10 @@ their default values.
 | `azureLandscape.tenantId`  | Id of Azure tenant to discover resources in |             |
 | `azureLandscape.subscriptions`  | List of Azure subscription ids to discover resources in | `[]`            |
 | `resourceDiscoveryGroups`  | List of resource discovery groups to configured following the [resource discovery declaration docs](https://promitor.io/configuration/v2.x/resource-discovery) |        |
-| `azureAuthentication.appId`  | Id of the Azure AD entity to authenticate with |             |
-| `azureAuthentication.appKey`  | Secret of the Azure AD entity to authenticate with |             |
```

Comment:
We'll have to be backward compatible here where the new one is favored over the old one; but we shouldn't break the current experience.

We can thus keep this but mark it as deprecated with reference to new appraoch